TransferList/TransferList.js

```diff
@@ -423,7 +473,7 @@ const TransferListBase = kind({
 			setNewList(list);
 		};
 
-		const rearrangeLists = (sourceList, destinationList, draggedElementIndex, draggedElementList, dragOverElementIndex, setSourceList, setDestinationList) => {
+		const rearrangeLists = useCallback((sourceList, destinationList, draggedElementIndex, draggedElementList, dragOverElementIndex, setSourceList, setDestinationList) => {
```

Comment:
Note: `rearrangeLists` was refactored to `useCallback` as a consequence of refactoring `onDropRightHandler` and `onDropLeftHandler` to use `useCalback`.
![image](https://user-images.githubusercontent.com/63335823/199952355-7255d8eb-fdc8-4773-8903-83b08034867a.png)